### ISSUE TITLE
[Suggestion] Leveraging vanilla MC chest placement

### ISSUE THREAD

**Issue by BLovegrove** [4nner/IronChests#3]
Saw nothing about where to post suggestions so I'll just post this here:

Now that you can place chests either apart or as a double chest in vanilla MC while the chests are next to each other it would be neat to see the same thing with Iron Chests.

**Comment by cyberanner** [comment]
@BLovegrove Are you suggesting to implement the possibility to have a Double Iron Chest, Double Diamond Chest, etc.?

**Comment by BLovegrove** [comment]
yes exactly that - essentially the exact same functionality as Minecraft chests as they are now in 1.17 but with the larger variants (copper, iron, gold, etc.)

**Comment by cyberanner** [comment]
Originally Iron Chests was meant to allow people to have single bigger chests (Like the iron chest being as big as a double chest by itself). I'd like to keep this original idea, and do not make double chests possible. However, I will not close this Issue/Feature Request, as if many people will request this feature to be implemented, I will consider to actually develop it and make it part of a future Iron Chests release.

**Comment by BLovegrove** [comment]
Neat - yeah I get where you're coming form there and I appreciate you keeping an open mind about potential features!

**Comment by vos6434** [comment]
Expanded Storage adds double chests.
https://www.curseforge.com/minecraft/mc-mods/expanded-storage-fabric

**Comment by 4nner** [comment]
Reopening to track interest.
Might implement it in the future.
